apps/storybook/.storybook/preview.tsx

```diff
@@ -1,4 +1,4 @@
-import '@digdir/designsystemet-css';
+import '@digdir/designsystemet-css/index.css';
```

Comment:
Huh, does this make a difference?

Reply:
CSS is never updated if we use the built css file. We need to use the one recieving updates for development to work without having to rebuild css every time we make changes